tools/xcodeproj_shims/install.sh

```diff
@@ -106,3 +106,4 @@ fi
 # which XCode will use for indexing. Let's keep those.
 "$BAZEL_INSTALLERS_DIR"/swiftmodules.sh >"$BAZEL_DIAGNOSTICS_DIR"/swiftmodules-stdout-"$DATE_SUFFIX".log 2>"$BAZEL_DIAGNOSTICS_DIR"/swiftmodules-stderr-"$DATE_SUFFIX".log &
 "$BAZEL_INSTALLERS_DIR"/indexstores.sh >"$BAZEL_DIAGNOSTICS_DIR"/indexstores-stdout-"$DATE_SUFFIX".log 2>"$BAZEL_DIAGNOSTICS_DIR"/indexstores-stderr-"$DATE_SUFFIX".log &
+"$BAZEL_INSTALLERS_DIR"/dsyms.sh >"$BAZEL_DIAGNOSTICS_DIR"/dsyms-stdout-"$DATE_SUFFIX".log 2>"$BAZEL_DIAGNOSTICS_DIR"/dsyms-stderr-"$DATE_SUFFIX".log &
```

Comment:
Any tradeoffs on making this a background job? I was wondering if that can introduce any race conditions in consumers of this: for instance if any program requires these to be present immediately proceeding a build.

I was also curious about how the ordering should stack up against these other jobs. I was wondering if its possible for the profiler to fail due to contention against a long index-import operation?

Reply:
I admit I interpreted this pattern as organizational in nature and not something that promoted async.

The profiler would only fail if the profiling task started before the dsyms were copied, which is technically possible but in practice nearly impossible (launching Instruments would take orders of magnitude longer).

If you think it's more appropriate to do this work directly in **install.sh**, synchronously, I'm totally able to move it up.